libs/tup-components/src/allocations/AllocationTable.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { LoadingSpinner, InlineMessage } from '@tacc/core-components';
+import { useProjects } from '@tacc/tup-hooks';
+import { formatDate } from '../utils/timeFormat';
+import styles from './Allocation.module.css';
+
+export const AllocationsTable: React.FC = () => {
+  const { data, isLoading, error } = useProjects();
+
+  if (isLoading) {
+    return <LoadingSpinner />;
+  }
+
+  if (error) {
+    return (
+      <InlineMessage type="warn">
```

Comment:
**Avoid Warning:** This change avoids the console warning `In a <(Section|Inline)Message> `type="warn"` is deprecated. Use ‵type="warning"‵ instead.`

```suggestion
      <InlineMessage type="warning">
```